src/dnddiceroller/Fudge.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dnddiceroller;
+
+import java.util.Random;
+
+/**
+ *
+ * @author Zachary
+ */
+public class Fudge implements Die {
+    private int sideOfDie;
+    private int valueOfDie;
+    
+    public int getvaleOfDie(){
```

Comment:
Not sure if this created any issues for you, but it looks like there may be a typo in this method.